far/FarEng.hlf.m4

```diff
@@ -1688,19 +1688,19 @@ separators, which by default are: #!%^&*()+|{}:"<>?`-=\\[];',./#.
 
     By checking the #Search for hex# option you can search for the files
 containing hexadecimal sequence of the specified bytes. In this case #Case#
-#sensitive#, #Whole words#, #Using code page# and #Search for folders#
+#sensitive#, #Whole words#, #Using codepage# and #Search for folders#
 options are disabled and their values doesn't affect the search process.
 
     #Not containing# allows to find files #not# containing the specified text or code.
 
-    Выпадающий список #Используя кодовую страницу# позволяет выбрать конкретную
-кодовую страницу, применяемую для поиска текста. Если в выпадающем списке выбрать
-пункт #Все кодовые страницы#, то Far будет использовать для поиска все стандартные
-(ANSI, OEM, UTF-8, UTF-16, UTF-16 BE) и #Избранные# кодовые страницы (список #избранных#
-кодовых страниц можно настроить в ~меню выбора кодовой страницы~@CodepagesMenu@
-настроек, редактора или программы просмотра). При необходимости поиска по собственному
-набору кодовых страниц отметьте их в списке при помощи клавиш #Ins# или #Space# и выберите
-пункт #Отмеченные кодовые страницы#.
+    Dropdown list #Using code page# allows to choose a specific codepage
+to be used for the search. If the item #All standard codepages# is selected
+in the dropdown list, Far will use all standard codepages (ANSI, OEM, UTF-8,
+UTF-16, UTF-16 BE), as well as #Favorite# codepages (the list of #favorite#
+codepages can be specified in the ~Codepages menu~@CodepagesMenu@
+in the options, editor, or viewer). To search using custom set of codepages,
```

Comment:
Shouldn't it be "using **a** custom set"?

Reply:
Of course it should! Will fix.